scripts/docs-content-link-rewrites/helpers/fetch-nav-data-for-repo-and-base-path.ts

```diff
@@ -5,6 +5,20 @@
 
 import fetchGithubFile from '@build-libs/fetch-github-file'
 
+export type fileContentEntry = {
+	title?: string
```

Comment:
Looking at these types of nav data, I think you'll need to add `heading`
```suggestion
	heading?: string
	title?: string
```